templates/header.ejs

```diff
@@ -13,6 +13,7 @@
         <link rel="stylesheet" href="css/bootstrap.min.css">
         <link rel="stylesheet" href="css/code-themes/sl-theme.css">
```

Comment:
i wonder if we still need the sl-theme.css

Reply:
It has some classes like "`.tomorrow-*`" but I'm not sure where they're used, so I think it's safer to keep it. My changes most overrode those in `bootstrap.min.css`.